ultravox/data/datasets.py

```diff
@@ -308,8 +318,11 @@ def _get_answer_messages(
         ]
 
     def _get_transcribe_messages(self, text: str) -> List[Dict[str, str]]:
+        prompt = self._get_transcribe_prompt()
+        if not self._args.include_audio:
+            prompt = prompt.replace("<|audio|>", text)
```

Comment:
Maybe put quotes around the text content so it ends up being like `Transcribe "Hello world"`

Reply:
Hmm, makes sense.
I'm wondering if we need this for audio as well then. Assuming perfect alignment, audio and text should be replaceable.

Reply:
@zqhuang211 do you have any strong opinions here? I.e., what is the right text version of this audio prompt?

Choices:
1. `Transcribe <|audio|>` ->  `Transcribe Hello world`
1. `Transcribe <|audio|>` ->  `Transcribe "Hello world"`
1. `Transcribe "<|audio|>"` ->  `Transcribe "Hello world"`

Or even:
1. `Transcribe\n<|audio|>` ->  `Transcribe\nHello world`

Reply:
I lean toward 4) as perhaps the cleanest approach.

Reply:
Switching all `Transcribe <|audio|>` to `Transcribe\n<|audio|>`.
I didn't switch "all" prompts since I felt it was good to have them in training still.